content/en/docs/architecture/ci-operator.md

```diff
@@ -533,6 +533,60 @@ The _released_ `4.Y` versions of `ClusterImageSet` [manifests](https://github.co
 a tool which ensures that they points to the latest version.
 It is currently not supported that a test claims by `cluster_claim` a cluster with the version which has not been released yet.
 
+### Testing with a Cluster from HyperShift
+The [hypershift-hostedcluster-workflow](https://steps.ci.openshift.org/workflow/hypershift-hostedcluster-workflow) is provided to claim a hosted cluster from [HyperShift](https://hypershift-docs.netlify.app/) deployed in the management cluster, which is another alternative to provision an OpenShift cluster for CI e2e tests.
+See the following test stanza as an example:
+
+```yaml
+...
+releases:
+  latest: # required; a dependency of the workflow to provide "release:latest"
+    release:
+      channel: stable
+      version: "4.12"
+- as: e2e-hypershift
+  steps:
+    cluster_profile: aws-2 # required; the cluster profile to create the cloud resources for the hosted cluster by HyperShift
+    env:
+      HYPERSHIFT_BASE_DOMAIN: hypershift.aws-2.ci.openshift.org # required; the basic domain for the hosted cluster
+      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.12.9-multi # required; the payload image to install on the host cluster
```

Comment:
The workflow should not need those two variables.
As discussed, let us take the default value from `BASE_DOMAIN` and `release:latest` and hide them from the users.
It would be harder to change after users use the workflow.
It will benefit the migration as well with the default values.

/hold

Reply:
Actually, it is already done that way for `RELEASE_IMAGE`
https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh#L23

just not for `DOMAIN`:

https://github.com/openshift/release/blob/ffa3eeaf0c3758f0f28f71e3e8002118d7a3b1cc/ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh#L17

Reply:
`RELEASE_IMAGE` does not work either:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/38225/rehearse-38225-pull-ci-openshift-cluster-autoscaler-operator-release-4.12-e2e-aws-operator/1645922231840149504

This makes the hosted cluster cannot be used for any core-component e2e tests.
Let us discuss it tomorrow.

Reply:
HYPERSHIFT_BASE_DOMAIN is fixed by https://github.com/openshift/release/pull/38304
@bear-redhat told me RELEASE_IMAGE is a known issue.